common/defs/bitcoin/zencash.json

```diff
@@ -1,16 +1,16 @@
 {
-  "coin_name": "Zencash",
+  "coin_name": "Horizen",
   "coin_shortcut": "ZEN",
-  "coin_label": "Zencash",
-  "website": "https://zencash.com",
+  "coin_label": "Horizen",
+  "website": "https://www.horizen.global",
   "github": "https://github.com/ZencashOfficial/zen",
-  "maintainer": "Zencash <support@zencash.com>",
+  "maintainer": "Horizen <support@horizen.global>",
```

Comment:
Please use your personal email, so everybody knows you are responsible for keeping this file up-to-date all the times.

Reply:
Done. Also renamed the file, deleted the old one and uploaded the new logo.